m/configDTPathsLocal.m

```diff
@@ -53,17 +53,19 @@
 %
 %  You should have received a copy of the GNU General Public License
 %  along with this program.  If not, see <http://www.gnu.org/licenses/>.
-
+%
+%  ** NOTE ** full path!! (no ~/...)    
+    
   error(nargchk(0, 0, nargin, 'struct'));
 
-  local_paths.binary_path    = '/path/to/delayed_time/glider_data/${GLIDER_NAME}/${DEPLOYMENT_START,Tyyyymmdd}/binary';
-  local_paths.cache_path     = '/path/to/delayed_time/glider_data/${GLIDER_NAME}/${DEPLOYMENT_START,Tyyyymmdd}/binary';
-  local_paths.log_path       = '/path/to/delayed_time/glider_data/${GLIDER_NAME}/${DEPLOYMENT_START,Tyyyymmdd}/log';
-  local_paths.ascii_path     = '/path/to/delayed_time/glider_data/${GLIDER_NAME}/${DEPLOYMENT_START,Tyyyymmdd}/ascii';
-  local_paths.figure_path    = '/path/to/delayed_time/glider_data/${GLIDER_NAME}/${DEPLOYMENT_START,Tyyyymmdd}/figures';
-  local_paths.netcdf_l0      = '/path/to/delayed_time/glider_data/${GLIDER_NAME}/${DEPLOYMENT_START,Tyyyymmdd}/netcdf/${GLIDER_NAME}_${DEPLOYMENT_START,Tyyyymmdd}_l0.nc';
-  local_paths.netcdf_l1      = '/path/to/delayed_time/glider_data/${GLIDER_NAME}/${DEPLOYMENT_START,Tyyyymmdd}/netcdf/${GLIDER_NAME}_${DEPLOYMENT_START,Tyyyymmdd}_l1.nc';
-  local_paths.netcdf_l2      = '/path/to/delayed_time/glider_data/${GLIDER_NAME}/${DEPLOYMENT_START,Tyyyymmdd}/netcdf/${GLIDER_NAME}_${DEPLOYMENT_START,Tyyyymmdd}_l2.nc';
-  local_paths.processing_log = '/path/to/delayed_time/glider_data/${GLIDER_NAME}/${DEPLOYMENT_START,Tyyyymmdd}/processing.log';
+  local_paths.binary_path    = '/home/cyrf0006/data/gliders_data/${GLIDER_NAME}/${DEPLOYMENT_START,Tyyyymmdd}/binary';
+  local_paths.cache_path     = '/home/cyrf0006/data/gliders_data/${GLIDER_NAME}/${DEPLOYMENT_START,Tyyyymmdd}/binary';
+  local_paths.log_path       = '/home/cyrf0006/data/gliders_data/${GLIDER_NAME}/${DEPLOYMENT_START,Tyyyymmdd}/log';
+  local_paths.ascii_path     = '/home/cyrf0006/data/gliders_data/${GLIDER_NAME}/${DEPLOYMENT_START,Tyyyymmdd}/ascii';
+  local_paths.figure_path    = '/home/cyrf0006/data/gliders_data/${GLIDER_NAME}/${DEPLOYMENT_START,Tyyyymmdd}/figures';
+  local_paths.netcdf_l0      = '/home/cyrf0006/data/gliders_data/${GLIDER_NAME}/${DEPLOYMENT_START,Tyyyymmdd}/netcdf/${GLIDER_NAME}_${DEPLOYMENT_START,Tyyyymmdd}_l0.nc';
+  local_paths.netcdf_l1      = '/home/cyrf0006/data/gliders_data/${GLIDER_NAME}/${DEPLOYMENT_START,Tyyyymmdd}/netcdf/${GLIDER_NAME}_${DEPLOYMENT_START,Tyyyymmdd}_l1.nc';
+  local_paths.netcdf_l2      = '/home/cyrf0006/data/gliders_data/${GLIDER_NAME}/${DEPLOYMENT_START,Tyyyymmdd}/netcdf/${GLIDER_NAME}_${DEPLOYMENT_START,Tyyyymmdd}_l2.nc';
+  local_paths.processing_log = '/home/cyrf0006/data/gliders_data/${GLIDER_NAME}/${DEPLOYMENT_START,Tyyyymmdd}/processing.log';
 
```

Comment:
I do not think that you want to upload your configuration as example values to the public repository.
You can revert this with `git reset master m/configDTPathsLocal.m`.
Your modifications will be preserved in the working directory.